howto/update/upgrade-anbox.md

```diff
@@ -58,7 +58,7 @@ The deployed Juju charms need to be upgraded next.
 
 - You can find a list of all charm, snap and Debian package versions for each Anbox Cloud release in the [component versions](https://discourse.ubuntu.com/t/component-versions/21413) overview. This also includes the charm and bundle revisions and channels for each release.
 
-- Starting with the 1.14 release, all charms come from [Charmhub](https://charmhub.io) and use the concept of [channels](https://snapcraft.io/docs/channels) to track particular versions. The instructions below address how to upgrade from a 1.13.x release, where charms were still from the old Juju charm store. The `--channel=1.21/stable` argument instructs Juju to switch to the latest [Charmhub](https://charmhub.io) version of the charm and track the right channel.
+- Starting with the 1.14 release, all charms come from [Charmhub](https://charmhub.io) and use the concept of [channels](https://snapcraft.io/docs/channels) to track particular versions. The instructions below address how to upgrade from a 1.13.x release, where charms were still from the old Juju charm store. The `--channel=1.22/stable` argument instructs Juju to switch to the latest [Charmhub](https://charmhub.io) version of the charm and track the right channel.
```

Comment:
A bit unrelated to the actual change you made, but do we still need that sentence anymore? 1.13 and 1.14 date back to 2022, two years ago. These are now long out of support.

Reply:
Hmm, I don't see any harm in removing it. Channels are being used for many releases now, it's enough to just retain the last statement about the `<latest_version>/stable` version tracking the right channel. Do you agree @adglkh @morphis ?

Reply:
Removed information about 1.14 and 1.15.